middleman-core/middleman-core.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency('dotenv')
   s.add_dependency('rgl', ['~> 0.5.3'])
   s.add_dependency('toml')
+  s.add_dependency('webrick') if RUBY_VERSION >= '3.0'
```

Comment:
I'm not sure this actually will work. I think its fine to include the dep for all versions

Reply:
I did test it using Bundler's path option, but you're right that it may behave differently with a released gem, and it looks like it may be causing the error with TravisCI too. 

I also don't see any strong reason for not including the dependency everywhere, so I've amended and rebased to always require it.